CP1404_practicals/prac_05/emails.py

```diff
@@ -13,10 +13,7 @@ def main():
             name = input("Please enter your name: ")
         email_information[user_email] = name
         user_email = input("Please enter your email: ")
-
-        # Display users names and email address neatly
-    for user_email, name in email_information.items():
-        print("{} --> ({})".format(name, user_email))
+    print(nice_display(email_information))
```

Comment:
Since the function 'nice_display' already has the print command included in it you could simplify this line by writing it as:
nice_display(email_information)